ledger/ledger-consensual/src/main/kotlin/net/corda/v5/ledger/consensual/transaction/ConsensualTransactionBuilder.kt

```diff
@@ -44,5 +44,5 @@ interface ConsensualTransactionBuilder {
      *      transactions accidentally.
      */
     @Suspendable
-    fun signInitial(publicKey: PublicKey): ConsensualSignedTransaction
+    fun sign(publicKey: PublicKey): ConsensualSignedTransaction
```

Comment:
Can we align the 'sign' methods with those on UTXO. ie. one that takes no parameters and signs what it can and one Iterable<PublicKey> and one vararg. We then don't need the singular version as the vararg covers it

Reply:
Done.